src/components/LoginPage/LoginPageComponent.js

```diff
@@ -1,67 +1,69 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import slide1 from 'components/LoginPage/images/slide-1.png';
-import slide2 from 'components/LoginPage/images/slide-2.png';
-import slide3 from 'components/LoginPage/images/slide-3.png';
-import slide4 from 'components/LoginPage/images/slide-4.png';
+import styled from 'styled-components';
+import Slide1 from 'components/LoginPage/Slide1';
+import Slide2 from 'components/LoginPage/Slide2';
+import Slide3 from 'components/LoginPage/Slide3';
+import Slide4 from 'components/LoginPage/Slide4';
+import Slide5 from 'components/LoginPage/Slide5';
+import iconGoogle from 'components/LoginPage/images/icon-google.svg';
+
+const Main = styled.main`
+  width: 100%;
+  background-color: #131c3e;
+  font-family: Roboto;
+`;
+
+const Article = styled.article`
+  width: 100%;
+  max-width: calc(100vh*1.1);
```

Comment:
Why `1.1`? What is `1.1` ? Please give it a name and reveal the formula, e.g.
```
const aspectRatio = 16 / 9;
```
```
const Article = styled.article`
  width: 100%;
  max-width: calc(100vh * aspectRatio)
```

Reply:
Done in commit [db8d5d2](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/db8d5d213e208ca30de77f1c6674c40053f31f87)